scene/gui/scroll_container.cpp

```diff
@@ -94,33 +94,35 @@ void ScrollContainer::gui_input(const Ref<InputEvent> &p_gui_input) {
 	Ref<InputEventMouseButton> mb = p_gui_input;
 
 	if (mb.is_valid()) {
+		// original sensitivity was 0.125
+		float scaled_sensitivity = sensitivity * 0.125;
```

Comment:
Maybe `sensitivity` should default to `0.125` instead of being a scaling factor for a hardcoded `0.125`?

Reply:
I think scaling it this way works well as having the default appear to the user as '1.0' makes the factor easy to understand in your head. But if you want me to remove the 0.125 scaling factor and change the default to 0.125 instead of 1.0 then I will do that.